### ISSUE TITLE
Data size limitation

### ISSUE THREAD

**Issue by LRydin** [LRydin/Power-Grid-Frequency-Data#3]
Currently this GitHub repository is already at 1GB, and we are still missing the British and Nordic data.
I've just realised that the Nordic data, as it has 0.1 seconds resolution, will likely occupy something close to 4~7GB, which is beyond the limit here in GitHub.

I might have to move the whole content to OpenScienceFramework (OSF), which has unlimited storage, is free, and cannot be removed, which I find the best option so far.

**Comment by philbox2** [comment]
Yeah, I think github is not meant to have large files. How about using [git-lfs](https://git-lfs.github.com/). Zenodo might also be a good option to make it citable.
OSF might also be a good option. I don't know much about it ;) 
Additionally, the [openmod-wiki](https://wiki.openmod-initiative.org/wiki/Data) might also be a good resource to read about what options there are and also all that licensing stuff.

**Comment by LRydin** [comment]
I was looking into git-lfs, but I couldn't figure out how to get direct links to download the files from there. The file is replaced by a pointer that then refers to the file in the LFS. I haven't figured out how to get a direct link so we can keep the same URL structure as `path/to/file/<location>/<year>/<month>/file_name` and then just add the `year` + `month` and download the files...

**Comment by LRydin** [comment]
After some deliberation, we have decided to entirely move the data to the OSF frame. This means we'll delete this repo entirely :).

See you all at https://github.com/LRydin/Power-Grid-Frequency/
